### PR TITLE
Add Python Solution for Milk Measurement (Bronze)

### PR DESCRIPTION
**Pull request by ryanchou-dev**
all test cases pass for this solution :)

### CODE REVIEW

File: solutions/usaco-761.mdx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+sys.stdin = open('measurement.in', 'r')
+sys.stdout = open('measurement.out', 'w')
+
+n = int(input())
+
+# Logs will store each entry in FJ's book.
+logs = []
+
+# Every cow starts off producing 7 gallons of milk each.
+bessie, elsie, mildred = 7, 7, 7
```
Comment (SansPapyrus683) [inline solutions/usaco-761.mdx:31]: magic number- consider replacing 7 with `INIT_MILK`

File: solutions/usaco-761.mdx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+		bessie += i[2]
+	elif i[1] == 'Elsie':
+		elsie += i[2]
+	elif i[1] == "Mildred":
+		mildred += i[2]
+
+    # Find the maximum amount of milk that's being produced.
+	curmax = max(bessie, max(elsie, mildred))
+
+    # Duplicate the current display, this will be used to
```
Comment (SansPapyrus683) [inline solutions/usaco-761.mdx:67]: indentation's kinda hecked?

File: solutions/usaco-761.mdx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+			display.remove('e')
+
+    # Same for Mildred.
+	if mildred == curmax:
+		display.add('m')
+	else:
+		if 'm' in display:
+			display.remove('m')
+    
+    # If the display has changed, FJ needs to update it.
```
Comment (SansPapyrus683) [inline solutions/usaco-761.mdx:95]: i like comments as much as the next coder
but this just seems a bit too bloated
here's a list of the unnecessary lines (including whitespace)
26, 27, 29, 45, 50, 52, 53, 55, 56 are the ones i see rn

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/usaco-761.mdx (hunk 3 above)
Comment (ryanchou-dev) [reply solutions/usaco-761.mdx:95]: 👍

File: solutions/usaco-761.mdx (hunk 2 above)
Comment (ryanchou-dev) [reply solutions/usaco-761.mdx:59]: wait what's wrong 🤔

File: solutions/usaco-761.mdx [hunk 4]
```diff
@@ -0,0 +1,94 @@
+---
+id: usaco-761
+source: USACO Bronze 2017 December
+title: Milk Measurement
+author: Ryan Chou
+---
+
+[Official Analysis (C++ and Java)](http://www.usaco.org/current/data/sol_measurement_bronze_dec17.html)
+
+## Solution
```
Comment (maggie-j-liu) [inline solutions/usaco-761.mdx:10]: maybe ## Explanation to be consistent?

File: solutions/usaco-761.mdx (hunk 2 above)
Comment (SansPapyrus683) [reply solutions/usaco-761.mdx:59]: image
idk if it looks that way on your machine but

Comment (maggie-j-liu) [reply solutions/usaco-761.mdx:59]: the comments are indented with spaces but the code is indented with tabs

Comment (ryanchou-dev) [reply solutions/usaco-761.mdx:59]: oh

**Review by maggie-j-liu — CHANGES_REQUESTED** [review]

**Comment by SansPapyrus683** [comment]
ryan i unironically simp for you and your code

File: solutions/usaco-761.mdx [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+[Official Analysis (C++ and Java)](http://www.usaco.org/current/data/sol_measurement_bronze_dec17.html)
+
+## Explanation
+
+Since there are only $3$ cows and $N \leq 1000$, we can sort all of the entries by date and update the selected cow's milk value for each entry.
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N)$
```
Comment (maggie-j-liu) [inline solutions/usaco-761.mdx:16]: n log n since you're sorting the logs?

File: solutions/usaco-761.mdx [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+for i in logs:
+	if i[1] == 'Bessie':
+		bessie += i[2]
+	elif i[1] == 'Elsie':
+		elsie += i[2]
+	elif i[1] == "Mildred":
+		mildred += i[2]
+
+	# Find the maximum amount of milk that's being produced.
+		curmax = max(bessie, max(elsie, mildred))
```
Comment (SansPapyrus683) [inline solutions/usaco-761.mdx:57]: uh what happened here

File: solutions/usaco-761.mdx (hunk 5 above)
Comment (ryanchou-dev) [reply solutions/usaco-761.mdx:16]: oops yes

File: solutions/usaco-761.mdx (hunk 6 above)
Comment (ryanchou-dev) [reply solutions/usaco-761.mdx:57]: wait need to change github's indents from 8 spaces to 4 spaces

Comment (SansPapyrus683) [reply solutions/usaco-761.mdx:57]: still seems to be weird?

File: solutions/usaco-761.mdx [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+	tmp_display = display.copy()
+
+	# If Bessie is producing the maximum,
+	if bessie == curmax:
+	# add her to the display.
+		display.add('b')
+	else:
+	# If she isn't producing the maximum and she's currently 
+	# in the display, remove her.
+		if 'b' in display:
```
Comment (SansPapyrus683) [inline solutions/usaco-761.mdx:70]: wouldn't just an elif work?

Comment (ryanchou-dev) [reply solutions/usaco-761.mdx:70]: like change `else:` to `elif bessie != curmax:`? think that might be unnecessary 🤷‍♂️

Comment (ryanchou-dev) [reply solutions/usaco-761.mdx:70]: cuz there's only one if statement in the first place

File: solutions/usaco-761.mdx (hunk 6 above)
Comment (ryanchou-dev) [reply solutions/usaco-761.mdx:57]: 🤔

File: solutions/usaco-761.mdx (hunk 7 above)
Comment (SansPapyrus683) [reply solutions/usaco-761.mdx:70]: no like `elif 'b' in display:`
bc rn it's an else than an if with nothing purely in the else

File: solutions/usaco-761.mdx (hunk 6 above)
Comment (SansPapyrus683) [reply solutions/usaco-761.mdx:57]: image

Comment (ryanchou-dev) [reply solutions/usaco-761.mdx:56]: it looks fine in the editor?
image

Comment (ryanchou-dev) [reply solutions/usaco-761.mdx:56]: i also indented the comment (that might have been the problem)

File: solutions/usaco-761.mdx (hunk 7 above)
Comment (ryanchou-dev) [reply solutions/usaco-761.mdx:69]: iirc you can't `elif` without an `if` in the same indentation level

Comment (nathangong) [reply solutions/usaco-761.mdx:69]: yeah i agree with sans

Comment (nathangong) [reply solutions/usaco-761.mdx:69]: it would be on the same indentation level as `if bessie == curmax:` right?

Comment (ryanchou-dev) [reply solutions/usaco-761.mdx:69]: but there's an `else:` after that
image

Comment (ryanchou-dev) [reply solutions/usaco-761.mdx:69]: and then the `if:` is in that `else:`

Comment (nathangong) [reply solutions/usaco-761.mdx:69]: sans is saying to replace the else with an elif like this:
```python
if bessie == curmax:
    display.add('b')
elif 'b' in display:
    display.remove('b')
```

File: solutions/usaco-761.mdx (hunk 6 above)
Comment (SansPapyrus683) [reply solutions/usaco-761.mdx:56]: wait is that curmax supposed to be in the elif?

File: solutions/usaco-761.mdx (hunk 7 above)
Comment (ryanchou-dev) [reply solutions/usaco-761.mdx:69]: im an idiot

File: solutions/usaco-761.mdx (hunk 6 above)
Comment (ryanchou-dev) [reply solutions/usaco-761.mdx:56]: no

**Comment by ryanchou-dev** [comment]
i think i addressed everything @SansPapyrus683 and @maggie-j-liu suggested. lmk if there's anything else :)

**Review by maggie-j-liu — APPROVED** [review]

**Review by nathangong — APPROVED** [review]

**Review by SansPapyrus683 — APPROVED** [review]

**Pull request merged**